lib/dma.c

```diff
@@ -167,7 +167,13 @@ MOCK_DEFINE(dma_controller_remove_region)(dma_controller_t *dma,
         }
 
         if (dma_unregister != NULL) {
+#ifdef DEBUG
+            dma->vfu_ctx->in_cb = CB_DMA_UNREGISTER;
+#endif
             dma_unregister(data, &region->info);
+#ifdef DEBUG
```

Comment:
still think you can drop the DEBUG here

Reply:
Just to be clear, you want the #ifdef dropped from everywhere we track the callback type, right? If so, what do we do with `quiesce_check_allowed`? Do we still abort even in non-debug mode?

Reply:
no, I think that makes sense to be DEBUG only still?